annotate/config.ily

```diff
@@ -111,6 +130,31 @@
    (question . "Question:")
    (todo . "TODO:"))
 
+
+
+%%%%%%%%%%%%%%%%%
+%%%% HTML options
+%%%%%%%%%%%%%%%%%
+
+% Annotation types for html text output
+\registerOption scholarly.annotate.export.html.labels
+#`((critical-remark . "Critical Remark")
+  (musical-issue . "Musical Issue")
+  (lilypond-issue . "Lilypond Issue>")
+  (question . "Question")
+  (todo . "TODO"))
+
+
+% Which props to print to html
+\registerOption scholarly.annotate.export.html.props
+  #`(type grob-location grob-type message)
+
+% Which stylesheet to link in html (just a proof of concept at the moment)
+\registerOption scholarly.annotate.export.html.css
+  #"annotate-styles.css"
+
+
```

Comment:
I think we need the option

```lilypond
\registerOption scholarly.annotate.export.html.full-document ##t
```

here. Of course it also has to be processed somewhere ...